core/connection-pool/src/main/java/datawave/core/common/cache/BaseTableCache.java

```diff
@@ -248,11 +245,6 @@ public Boolean call() throws Exception {
 
     public void setupScanner(BatchScanner scanner) {
         scanner.setRanges(Lists.newArrayList(new Range()));
-        Map<String,String> options = new HashMap<>();
-        options.put(RegExFilter.COLF_REGEX, "^f$");
-        options.put("negate", "true");
-        IteratorSetting settings = new IteratorSetting(100, "skipFColumn", RegExFilter.class, options);
-        scanner.addScanIterator(settings);
```

Comment:
This should be behind a flag so that only the exact callers that need the F column get it. Otherwise all callers will start pulling far more data than necessary.

Reply:
Yeah, I think this is the wrong approach to the problem.  I am going to rework this to cache the index holes instead of caching the entire metadata table.

Reply:
used a different approach.  see updates.